### ISSUE TITLE
Assignment 2 - the format of y

### ISSUE THREAD

**Issue by Gitsamshi** [CISC-873/Information#8]
Hi prof, I have a small question for A2

In the second to last line of A2 code template, we convert y_test to int: y_test_df['infection'] = y_test.astype(int). But as we use AUROC as a metric on the predicted probability, Is it possible if we change it to  y_test_df['infection'] = y_test.astype(float)?

**Comment by steven-hh-ding** [comment]
@Gitsamshi  yes you are right. it should be as float! will fix today! (or if you don't mind to create a pull-request)
